src/Components/RadioButton/radioButton.css

```diff
@@ -0,0 +1,38 @@
+/** @format */
+.radio {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	cursor: pointer;
+	margin-right: 10px;
+}
+
+.radio input {
+	display: none;
+}
+
+.checkMark {
+	width: 1em;
+	height: 1em;
+	border: 2px solid #d8e4e2;
```

Comment:
Define this as a variable in the common variables file and access it as a variable.